packages/core/src/js/colors.ts

```diff
@@ -147,7 +138,7 @@ export const colorsBackgroundLight = {
 }
 
 export const colorsBackgroundUtility = {
-  base: '#8A99A8',
+  base: '138,153,168',
```

Comment:
Having some values rgba()-wrapped and some with raw csv will lead to unpredictable usage of these values. I would rather do them all at once and cut over. It's already a breaking change. 

What do you think?

Reply:
This is the only color we really need these raw values for.  because of how often we combine it with transparentize. The other colors are more one offs.